diff --git a/dist/src/utils/address-utils.js b/dist/src/utils/address-utils.js
index cfd666cbb5fc449313b42c1240fe32b59ae1632d..519a7ac9bb2084b4f35147f81d043a16a4691886 100644
--- a/dist/src/utils/address-utils.js
+++ b/dist/src/utils/address-utils.js
@@ -2,9 +2,11 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.makeRandomAddress = void 0;
 const utils_1 = require("ethers/lib/utils");
-const web3_utils_1 = require("web3-utils");
+const crypto = require("crypto");
+// const web3_utils_1 = require("web3-utils");
+const genRanHex = size => "0x"+ crypto.randomBytes(size).toString('hex');
 const makeRandomAddress = () => {
-    return (0, utils_1.getAddress)((0, web3_utils_1.randomHex)(20));
+    return (0, utils_1.getAddress)(0, genRanHex(20));
 };
 exports.makeRandomAddress = makeRandomAddress;
 //# sourceMappingURL=address-utils.js.map