diff --git a/node_modules/@defi-wonderland/smock/dist/src/utils/address-utils.js b/node_modules/@defi-wonderland/smock/dist/src/utils/address-utils.js
index cfd666c..6d1f3bb 100644
--- a/node_modules/@defi-wonderland/smock/dist/src/utils/address-utils.js
+++ b/node_modules/@defi-wonderland/smock/dist/src/utils/address-utils.js
@@ -1,10 +1,11 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.makeRandomAddress = void 0;
-const utils_1 = require("ethers/lib/utils");
-const web3_utils_1 = require("web3-utils");
+'use strict'
+Object.defineProperty(exports, '__esModule', { value: true })
+exports.makeRandomAddress = void 0
+const utils_1 = require('ethers/lib/utils')
+const crypto = require('crypto')
+const genRanHex = size => '0x' + crypto.randomBytes(size).toString('hex')
 const makeRandomAddress = () => {
-    return (0, utils_1.getAddress)((0, web3_utils_1.randomHex)(20));
-};
-exports.makeRandomAddress = makeRandomAddress;
+  return (0, utils_1.getAddress)(genRanHex(20))
+}
+exports.makeRandomAddress = makeRandomAddress
 //# sourceMappingURL=address-utils.js.map
