diff --git a/dist/src/utils/address-utils.js b/dist/src/utils/address-utils.js
index cfd666cbb5fc449313b42c1240fe32b59ae1632d..7e109d8b7de282e655b07df64b4da39aa4402f4e 100644
--- a/dist/src/utils/address-utils.js
+++ b/dist/src/utils/address-utils.js
@@ -2,9 +2,10 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.makeRandomAddress = void 0;
 const utils_1 = require("ethers/lib/utils");
-const web3_utils_1 = require("web3-utils");
+// const web3_utils_1 = require("web3-utils");
+const genRanHex = size => [...Array(size)].map(() => Math.floor(Math.random() * 16).toString(16)).join('');
 const makeRandomAddress = () => {
-    return (0, utils_1.getAddress)((0, web3_utils_1.randomHex)(20));
+    return (0, utils_1.getAddress)(genRanHex(20));
 };
 exports.makeRandomAddress = makeRandomAddress;
 //# sourceMappingURL=address-utils.js.map